src/screens/UserManagement/ShowUserData.res

```diff
@@ -35,23 +166,52 @@ module UserHeading = {
         isTag=true
         tagText={infoValue.role_name->String.toUpperCase}
       />
-      <div className="flex items-center gap-4">
-        <div className={`font-semibold text-green-700`}>
-          {switch status {
-          | InviteSent => "INVITE SENT"->String.toUpperCase->React.string
-          | _ => infoValue.status->String.toUpperCase->React.string
-          }}
-        </div>
-        <UIUtils.RenderIf condition={status !== Active}>
+      <RenderIf condition={isUpdateRoleSelected}>
+        <div className="flex items-center gap-2">
           <Button
-            text="Resend Invite"
-            buttonState
-            buttonType={SecondaryFilled}
-            customButtonStyle="!px-2"
-            onClick={_ => resendInvite()->ignore}
+            buttonType={Secondary}
+            text="Back"
+            onClick={_ => {
+              if newRoleSelected === infoValue.role_id {
+                setIsUpdateRoleSelected(_ => false)
+              } else {
+                updatePermissionInfoOnBack()->ignore
+              }
```

Comment:
Can we make a different function which handles this logic ?

Reply:
Updated